{"id":885,"details":"\n  \n  \u003ctable\u003e\n    \u003ctr\u003e\n      \u003ctd\u003eRegion\u003c/td\u003e\n\u003ctd\u003eAuvergne-RhÌ«ne-Alpes\u003c/td\u003e\n    \u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eDepartment\u003c/td\u003e\n\u003ctd\u003eHaute-Savoie\u003c/td\u003e\n\u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eCity\u003c/td\u003e\n\u003ctd\u003eLa Balme-de-Sillingy\u003c/td\u003e\n\u003c/tr\u003e\n    \n  \u003c/table\u003e\n  ","patient":"Patient From Auvergne Rhì«Ne Alpes, Haute Savoie, La Balme De Sillingy","source":"unknown_source","status":"in_hospital","confirmed_at":"2020-03-03T00:00:00.000Z","created_at":"2020-03-16T10:13:03.855Z","updated_at":"2020-03-16T10:42:30.202Z","infected_by":"unknown_infection","number":611,"urls":["https://github.com/lperez31/coronavirus-france-dataset"],"recovered_at":null,"gender":"unidentified_gender","age":0,"nationality":"unidentified_nationality","symptomatic_at":null,"slug":"france-case-611-patient-from-auvergne-rhi-ne-alpes-haute-savoie-la-balme-de-sillingy","suggested_url":"france-case-611-patient-from-auvergne-rhi-ne-alpes-haute-savoie-la-balme-de-sillingy","hospital_stay_length_in_days":null,"days_from_symptoms_to_confirmation":null,"without_symptoms":false,"reporting_country":"fr"}